src/Racing.js

```diff
@@ -0,0 +1,64 @@
+import Display from './Display.js';
+
+class Racing {
+  #totalRounds;
+
+  #cars;
+
+  #result;
+
+  constructor(totalRounds, cars) {
+    this.#totalRounds = totalRounds;
+    this.#cars = cars;
+    this.#result = [];
+  }
+
+  #tryRound() {
+    const cars = [...this.#cars];
+    const roundResult = cars.map((car) => car.driving());
+    this.#result.push(roundResult);
+  }
+
+  play() {
+    const totalRounds = this.#totalRounds;
+    const rounds = Array.from(
+      { length: totalRounds },
+      (arrayLike, index) => index + 1,
+    );
+
+    rounds.forEach(() => {
+      this.#tryRound();
+    });
+  }
+
+  showResult() {
+    const raceResult = this.#result.map((result) => [...result]);
+    raceResult.forEach((roundResult, index) => {
+      const round = index + 1;
+      Display.showRoundResultHeader(round);
+      Display.showRoundResult(roundResult);
+    });
+  }
```

Comment:
`showRoundResultHeader`인 경우 round가 1일때만 header를 출력하는 기능을 가지고 있더군요! 
다만 header가 출력된 이후에는 다시 재사용이 안되기 때문에 불필요하게 반복문을 진행할 것 같다는 생각이 들어요.
파라미더를 지우고 아래와 같이 foreach문을 돌기 전 한번만 호출하는 방식은 어떨까요? 처음에 의도하신 바가 있으실 것 같아 조심스레 여쭤봅니다

```suggestion
  showResult() {
    const raceResult = this.#result.map((result) => [...result]);
    Display.showRoundResultHeader();
    raceResult.forEach((roundResult, index) => {
      const round = index + 1;
      Display.showRoundResult(roundResult);
    });
  }
```

Reply:
오 `Display.showRoundResultHeader()`를 `forEach`문 밖으로 꺼내볼 생각은 미처 하지 못했네요. round를 계산할 필요도 없고, 문경님께서 말씀해주신 방법이 더 효율적인 것 같습니다. 제안 감사합니다!